### ISSUE TITLE
Reports stuck in processing

### ISSUE THREAD

**Issue by jethror1** [ewels/MegaQC#333]
**Question**

Sorry for more issues but we seem to be having a problem processing reports, I tried importing ~160 JSONs at once and I assumed it would queue them up and process them one at a time. I left this since yesterday and none seem to have been processed:

![image](https://user-images.githubusercontent.com/45037268/146356687-30c45154-1f75-425a-9b44-dcde951e21ad.png)

![image](https://user-images.githubusercontent.com/45037268/146356730-2bd08deb-9177-47a4-81e7-5f8708f94168.png)

Is there a way to see what is happening / why they haven't processed? Is there an issue with doing everything in one go and does it need an interval adding between uploads?

**Comment by multimeric** [comment]
Yes, this is a known problem actually that I've never remember to file as an issue. When the user posts a report it gets added to the database unprocessed, but then there's [a scheduler](https://github.com/ewels/MegaQC/blob/master/megaqc/scheduler.py) that runs, I think in another process using the [APScheduler](https://apscheduler.readthedocs.io/en/3.x/) library, which processes them. I think it should be possible to debug the parsing script by setting a breakpoint in it and restarting megaqc. Try that and let me know why they aren't getting finished.
